### PR TITLE
Revise docs

### PR DESCRIPTION
**Pull request by willthefrog**
- README.md
- GETTING_STARTED.md
- INSTALL.md
- DATA.md
- add CONFIG.md
- update ipython demo
- pick better images

### CODE REVIEW

File: PaddleCV/object_detection/README.md [hunk 1]
```diff
@@ -75,31 +76,38 @@ python tools/infer.py -c configs/mask_rcnn_r50_1x.yml \
...
-## Updates
 
-The major updates are as follows:
+## Updates
 
-#### 2019-07-03
-- Release the unified detection framework.
-- Supported algorithms: Faster R-CNN, Mask R-CNN, Faster R-CNN + FPN, Mask R-CNN + FPN, Cascade-Faster-RCNN + FPN, RetinaNet, Yolo v3, and SSD.
-- Release the first version of Model Zoo.
+#### v0.1 (7/3/2019)
```
Comment (qingqing01) [inline PaddleCV/object_detection/README.md]: Do not add v0.1? Since there is no version num now.

**Review by qingqing01 — APPROVED** [review]

File: PaddleCV/object_detection/README.md (hunk 1 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md]: then what should I put there?

Comment (willthefrog) [reply PaddleCV/object_detection/README.md]: changed to "initial release"

File: PaddleCV/object_detection/README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # PaddlePaddle Object Detection
 
-This object detection framework is based on PaddlePaddle. We want to provide classically and state of the art detection algorithms in generic object detection and specific target detection for users. And we aimed to make this framework easy to extend, train, and deploy. We aimed to make it easy to use in research and products.
+The goal of PaddleDetection is to provide easy access to a wide range of object
+detection models in both industry and research settings. we design
+PaddleDetection to be not only performant, production-ready but also highly
+flexible, catering to research needs.
 
 
 <div align="center">
```
Comment (willthefrog) [inline PaddleCV/object_detection/README.md:9]: needs better images

File: PaddleCV/object_detection/README.md [hunk 3]
```diff
@@ -10,63 +13,61 @@ This object detection framework is based on PaddlePaddle. We want to provide cla
...
-| Yolov3             | ✓      | ✗      | ✗         |  ✗       | ✓        | ✓     |
-| SSD                | ✗      | ✗      | ✗         |  ✗       | ✓        | ✗      |
+|                    | ResNet | ResNet-vd <sup>[1](#vd)</sup> | ResNeXt | SENet | MobileNet | DarkNet |
+|--------------------|:------:|------------------------------:|:-------:|:-----:|:---------:|:-------:|
+| Faster R-CNN       | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Faster R-CNN + FPN | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Mask R-CNN         | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Mask R-CNN + FPN   | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Cascade R-CNN      | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| RetinaNet          | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
```
Comment (FlyingQianMM) [inline PaddleCV/object_detection/README.md]: RetinaNet目前只支持ResNet，其他的先打成x吧

Comment (willthefrog) [reply PaddleCV/object_detection/README.md]: done

File: PaddleCV/object_detection/README.md [hunk 4]
```diff
@@ -10,63 +13,61 @@ This object detection framework is based on PaddlePaddle. We want to provide cla
...
-   All components are modular encapsulated, including the data transforms. It's easy to plug in and pull out any module. For example, users can switch backbone easily or add mixup data augmentation for models.
+- Highly Flexible:
+Components are designed to be modular. Model architectures, as well as data
+preprocess pipelines, can be easily customized with simple configuration
+changes.
 
-- High Efficiency:
-  Based on the high efficient PaddlePaddle framework, less memory is required. For example, the batch size of Mask-RCNN based on ResNet50 can be 5 per Tesla V100 (16G). The training speed of Yolo v3 is faster than other frameworks.  
+- Performance Optimized:
+with the help of the underlying PaddlePaddle framework, faster training and
```
Comment (qingqing01) [inline PaddleCV/object_detection/README.md]: with the help -> With the help

File: PaddleCV/object_detection/README.md [hunk 5]
```diff
@@ -10,63 +13,61 @@ This object detection framework is based on PaddlePaddle. We want to provide cla
...
+<a name="vd">[1]</a> ResNet-vd models offer much improved accuracy with negligible performance cost.
 
-- [x] **Synchronized batch norm**:  used in Yolo v3.
-- [x] **Group Norm**: supported this operation, the related model will be added later.
-- [x] **Modulated deformable convolution**: supported this operation, the related model will be added later.
-- [x] **Deformable PSRoI Pooling**: supported this operation, the related model will be added later.
+Advanced Features:
 
+- [x] **Synchronized batch norm**: currently used by Yolo V3.
+- [x] **Group Norm**: pretrained models to be released.
```
Comment (qingqing01) [inline PaddleCV/object_detection/README.md:52]: Group Norm -> Group norm

File: PaddleCV/object_detection/README.md [hunk 6]
```diff
@@ -10,63 +13,61 @@ This object detection framework is based on PaddlePaddle. We want to provide cla
...
+- [x] **Modulated deformable convolution**: pretrained models to be released.
+- [x] **Deformable PSRoI Pooling**: pretrained models to be released.
```
Comment (qingqing01) [inline PaddleCV/object_detection/README.md:54]: Deformable PSRoI Pooling -> Deformable PSRoI pooling

File: PaddleCV/object_detection/docs/INSTALL.md [hunk 7]
```diff
@@ -60,23 +62,24 @@ To train the model, COCO-API is needed. Installation is as follows:
...
+with the following commands:
 
 ```
 cd <path/to/clone/models>
 git clone https://github.com/PaddlePaddle/models
 cd models/PaddleCV/object_detection
 ```
 
-**Install python module requirements:**
+**Install python dependencies:**
```
Comment (qingqing01) [inline PaddleCV/object_detection/docs/INSTALL.md]: python -> Python

File: PaddleCV/object_detection/README.md (hunk 5 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md:52]: on the contrary, I think they should be capitalized...

File: PaddleCV/object_detection/README.md (hunk 4 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md]: done

File: PaddleCV/object_detection/README.md (hunk 6 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md:54]: capitalized

File: PaddleCV/object_detection/README.md (hunk 5 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md:52]: capitalized

File: PaddleCV/object_detection/docs/INSTALL.md (hunk 7 above)
Comment (willthefrog) [reply PaddleCV/object_detection/docs/INSTALL.md]: done

File: PaddleCV/object_detection/docs/INSTALL.md [hunk 8]
```diff
@@ -86,18 +89,22 @@ python ppdet/modeling/tests/test_architectures.py
...
 
 ```
-ln -sf <path/to/coco> $PaddleDetection/data/coco
-ln -sf <path/to/voc> $PaddleDetection/data/voc
+ln -sf <path/to/coco> <path/to/paddle_detection>/data/coco
+ln -sf <path/to/voc> <path/to/paddle_detection>/data/voc
 ```
 
-If you do not have datasets locally, you can download dataset as follows:
+**Download datasets manually:**
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/INSTALL.md:105]: this part may need to be rebased...

File: PaddleCV/object_detection/README.md (hunk 5 above)
Comment (heavengate) [reply PaddleCV/object_detection/README.md:52]: pretrained models to be released.  -> supported, pretrained models to be released. ?

File: PaddleCV/object_detection/README.md [hunk 9]
```diff
@@ -10,63 +13,61 @@ This object detection framework is based on PaddlePaddle. We want to provide cla
...
-- About Algorithms:
-   - More SOTA models.
-   - More easy-to-deployed models.
-
-We are glad to receive your feedback.
 
 ## Model zoo
 
-The trained models can be available in PaddlePaddle [detection model zoo](docs/MODEL_ZOO.md).
+Pretrained models are available in the PaddlePaddle [detection model zoo](docs/MODEL_ZOO.md).
```
Comment (heavengate) [inline PaddleCV/object_detection/README.md:59]: [detection model zoo] -> [model zoo] ?

File: PaddleCV/object_detection/README.md (hunk 5 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md:52]: IMHO, putting them in the "advanced features" section implies that they are supported, we only need to clarify that the pretrained models are not available at the moment.

File: PaddleCV/object_detection/README.md (hunk 9 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md:59]: are these separated? @qingqing01

File: PaddleCV/object_detection/README.md (hunk 2 above)
Comment (willthefrog) [reply PaddleCV/object_detection/README.md:9]: done

File: PaddleCV/object_detection/README.md (hunk 9 above)
Comment (qingqing01) [reply PaddleCV/object_detection/README.md:59]: both ok

File: PaddleCV/object_detection/docs/CONFIG.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
+# Introduction
+
+PaddleDetection takes a rather principled approach to configuration management. We aim to automate the configuration workflow and to reduce configuration errors.
+
+
+# Rationale
+
+Presently, configuration in mainstream frameworks are usually dictionary based: the global config is simply a giant, loosely defined python dictionary.
```
Comment (qingqing01) [inline PaddleCV/object_detection/docs/CONFIG.md]: python -> Python

**Review by heavengate — APPROVED** [review]
LGTM

**Comment by willthefrog** [comment]
revised `DATA.md`, please take a look. @FlyingQianMM  @qingqing01

File: PaddleCV/object_detection/docs/INSTALL.md (hunk 8 above)
Comment (willthefrog) [reply PaddleCV/object_detection/docs/INSTALL.md:105]: done

**Review by qingqing01 — APPROVED** [review]

File: PaddleCV/object_detection/docs/CONFIG.md (hunk 10 above)
Comment (willthefrog) [reply PaddleCV/object_detection/docs/CONFIG.md]: done

**Pull request merged**